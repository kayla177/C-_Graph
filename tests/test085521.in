ENTITY 1A6YIJSN name_1A6Y institution
ENTITY BZM4OYTU name_BZM4 paper
ENTITY VWKK8B71 name_VWKK conference
RELATIONSHIP 1A6YIJSN affiliated_with BZM4OYTU 1.7
PRINT 1A6YIJSN
HIGHEST
EXIT
