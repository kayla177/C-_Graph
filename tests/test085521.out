success
success
success
success
BZM4OYTU
1A6YIJSN BZM4OYTU 1.7
